plugin/src/main/java/net/dzikoysk/funnyguilds/guild/GuildUtils.java

```diff
@@ -169,35 +141,25 @@ public static boolean tagExists(String tag) {
                 .collect(Collectors.toList());
     }
 
-    public static Set<Guild> getGuilds(Collection<String> names) {
-        return names.stream()
-                .map(GuildUtils::getByName)
-                .filter(Objects::nonNull)
-                .collect(Collectors.toSet());
-    }
-
-    public static synchronized void addGuild(Guild guild) {
-        if (guild == null) {
-            return;
-        }
+    public static void spawnHeart(PluginConfiguration pluginConfiguration, Guild guild) {
```

Comment:
Trochę kontrowersyjnie wkładać ciągle PluginConfiguration do funkcji

Reply:
@dzikoysk co ty uważasz bo ja się nie znam

Reply:
A czemu to nie może być po prostu metoda w GuildManagerze, tylko musi być w utilsach

Reply:
W sumie to może :lenny: